widget/edit.html

```diff
@@ -0,0 +1,171 @@
+<form data-ng-submit="save()" class="noMargin" name="editFunnelWidgetForm" data-ng-class="{'state-wait': processing }"
+    novalidate>
+    <div class="modal-header">
+        <h3 class="modal-title col-md-9">Edit Funnel Chart</h3>
+        <button type="button" class="close" data-ng-click="cancel()" data-dismiss="modal" aria-label="Close"
+            id="close-edit-widget-form-btn">
+            <div aria-hidden="true" class="version-button">+</div>
+        </button>
+    </div>
+    <div class="modal-body">
+        <div>
+            <div class="form-group"
+                data-ng-class="{ 'has-error': editFunnelWidgetForm.title.$invalid && editFunnelWidgetForm.title.$touched }">
+                <label for="title" class="control-label">Title<span class="text-danger">*</span></label>
+                <input id="title" name="title" type="text" data-ng-model="config.title" class="form-control" />
+                <div data-cs-messages="editFunnelWidgetForm.title"></div>
+            </div>
+            <div
+                data-ng-class="{ 'has-error': editFunnelWidgetForm.funnelModuleType.$invalid && editFunnelWidgetForm.funnelModuleType.$touched }">
+                <label for="funnelModuleType" class="control-label">Select Module Type</label><span
+                    class="text-danger">*</span>
+                <select name="funnelModuleType" id="funnelModuleType" class="form-control" style="width:30%"
+                    data-ng-options="key as value for (key, value) in funnelModuleType.type"
+                    data-ng-model="config.funnelModuleType" data-ng-change="onChangeModuleType()" required>
+                    <option value="">Select an Option</option>
+                </select>
+            </div>
+            <div data-ng-if="config.funnelModuleType == 0">
+                <div id="funnelModuleType0" class="form-group" data-ng-if="modules">
+                    <div data-ng-repeat="item in config.layers"class="form-group dashed-border-section" style="position:relative;">
+                        <div  style="padding: 5px;">
+                            <h6>Layer {{$index +1}}</h6>
+                            <label class="control-label">Title<span class="text-danger">*</span></label>
+                            <input name="layer" type="text" data-ng-model="item.title" class="form-control" required />
```

Comment:
![Screenshot 2023-02-17 at 12 07 39 PM](https://user-images.githubusercontent.com/39477678/219568165-9932e08e-e9e4-4db2-859f-106099554d4a.png)


Add some padding left and right